firedrake/mg/kernels.py

```diff
@@ -248,7 +291,7 @@ def prolong_kernel(expression):
                     break;
                 }
 
-                %(compute_celldist)s
+                %(compute_celldist_mg)s
```

Comment:
Yeah, the following code suggests to me that the "cell distance" computation that we're currently doing is complete nonsense and this should just be fixed to do something sane.